src/blocks/social/index.php

```diff
@@ -93,13 +93,16 @@ function coblocks_render_social_block( $attributes ) {
 	// Attributes.
 	$text_align    = is_array( $attributes ) && isset( $attributes['textAlign'] ) ? "style=text-align:{$attributes['textAlign']}" : false;
 	$border_radius = is_array( $attributes ) && isset( $attributes['borderRadius'] ) ? "border-radius: {$attributes['borderRadius']}px;" : false;
+	$has_padding   = is_array( $attributes ) && isset( $attributes['padding'] ) ? 'has-padding' : false;
```

Comment:
@rtabor Technically, when these checks (`is_array( $attributes ) && isset( $attributes['padding'] )`) don't return true, shouldn't these be empty strings instead of boolean values?

Reply:
Good catch! 🎣